dashboard/templates/registration/login.html

```diff
@@ -9,4 +9,5 @@ <h2>Login</h2>
     {{ form.as_p }}
     <button type="submit">Login</button>
   </form>
+  {% include "ga/include_ga.html" %}
```

Comment:
Here are my thoughts on this, the `login.html` is just for local development purposes and in the dev/prod. The trigger for Google analytics starts from here and I don't know if the tracking will start taking effect in higher env. I need to test this.

Reply:
I've removed the tracking from the login page for now since it will also be rewritten in react (just mentioned he had a branch for it)